com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/flowable/FlowableFacade.java

```diff
@@ -354,4 +368,22 @@ public boolean isJobExecutorActive() {
     public ProcessEngine getProcessEngine() {
         return processEngine;
     }
+
+    @SuppressWarnings("unchecked")
+    public <T> T getVariableFromHistoricVariables(List<HistoricVariableInstance> historicVariablesForProcess, String varName) {
```

Comment:
This method should not be necessary after applying my suggestion for `findHistoricVariablesForProcess`.